### ISSUE TITLE
Unisys Weekly Standups

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#2486]
Document any important notes from the weekly standups with Unisys.

**Comment by TracyLinder** [comment]
### 4/28/20

Paloma & team started testing the map viewer and the PSM folder last week. Unisys has been managing the feedback. 
- [ ] Tracy needs to find out what current priorities are (Backlog vs ROWMAN 4)
- [ ] Get finance approval of PSM fees
- [ ] David will schedule a meeting next week to look at current SOWs

SCP3 - Bugs have been resolved & is ready for UAT again. 
- [ ] Schedule UAT w/ April

**Comment by TracyLinder** [comment]
### 5/5/20

- SCP3 UAT should be done by end of week (April, Tracy, Nate)
- Discussed [this](https://trello.com/c/CC0zkpSv/12-141-correction-of-revision-required) SCP issue. I am concerned after performing UAT and seeing no noticeable changes made to correct the bugs reported in Jan. 
- David asked if the Payment Failure Comment Feature should be added to PSM. I said yes, following exactly how the other ROW folders do it. 
- The GIS errors that Paloma+team reported could not be duplicated by Unisys or Sean w/ COA. They aren't sure what happened, so Matt w/ Unisys is looking at it. David will follow up with him. 

**Overdue**
- [ ] Get finance approval of PSM fees

**Comment by TracyLinder** [comment]
- Clickmap: Matt found that not all java libraries were restored. Paloma know it's ready for testing again (email 5/6)
- PSM - Matt just added the payment error functionality. He will promote that through to DEV. Waiting on CTM to promote. Unisys will test in DEV, then Paloma's team will test in TEST. PSM fees still need finance approval. 
- Self assignment: Tracy still needs to schedule UAT assistance w/ Arthur
- ROWMAN 4: They have angular developers on staff. The estimates will probably need to be redone to account for angular development instead of smarguide. We shouldn't have to wait for the V7 upgrade to be complete; the code will migrate well. 
- SCP3: David went through test scenarios in DEV, and it's working. They're finding something in test was rolled back. They're looking to promote them back up after they find them all. Hoping to have it ready for CTM by tomorrow. 

- [ ] Tracy: Send David as much info about LIT Portal solution as Raj is comfortable with.

**Comment by TracyLinder** [comment]
### 5/26

- [ ] Ask Arthur/Melissa if they want Click Map on the LM application. 
- [ ] Post Payment bugs - are they working in test? If so lets get them promoted. (Me, check with PLo's team)
- [ ] Tracy - schedule time to test self assignment with Arthur 
- [ ]  David - Send tracy list of all PSM fees

**Comment by TracyLinder** [comment]
### 6/2

**SCP**
- David met w/ CTM yesterday about SCP. The split fee has to go to prod before or with SCP3 work. SCP3 work is dependent on the fee fix. 
    - Tracy asked RAM to help w/ getting SCP fee fix pushed through
- Tech issues blocked all promotes for the last 3-4 days. So still waiting on SCP3 bug fixes to get pushed up to test. He's hoping that happens today or tomorrow; will email us when we can begin testing again. 

**ROWMAN 2:** 
- Click map added to UC & PSM, not in test yet only in Dev. David will let us know when it's ready in test. 
- PSM: Two active issues. 
1. Having hard time generating the permit report, can this be retested? It's working just fine for David. 
    - [ ] Tracy: Do a quick test to see if it works for me
2. When Paloma put a zero in parking spaces info, no fees are added. It's an optional text field; we can leave as is, and reject anything that comes in w/ a zero, or leave as text but make it mandatory, or change it to a drop down with options being 0, 1, 2, 3. This seems to only be an issue for the patio subtype. **See David's email about this yesterday.**
    - [ ] Tracy: this issue needs to be better defined. 

**ROWMAN 3:** 
- Post Payment: Tracy, Kim, and Paloma had a testing work session last week. 
    - [ ] Tracy: Finish updated issues for the remaining bugs & send email update to team
- Self Assign: LM testing tomorrow w/ Arthur. We should see movement on that next week.

**Comment by TracyLinder** [comment]
### 6/9

**Tracy:** 
- All my tasks from last week are still outstanding. I blocked off time to work on them tomorrow 6/10 @ 3pm. 

**ROWMAN 2**
- Click map on UC, LM, & PSM are now in test 🎉
- Tracy to do PSM tasks tomorrow afternoon

**ROWMAN 3**
- One remaining issue for LM (remove duplicate issues)
- Tracy is testing w/ UC Jaclyn today @ 3

**ROWMAN 4**
- David going to work on updating the SOW w/ feedback provided by COA. Will need to update once the portal solution is in place, to better estimate how much effort is involved in developing those web pages. We haven't paid for the scoping exercises we did, I told David to send the invoice over. 

**SCP3** 
- Promoted to test. They're verifying it worked, then will let us know ok to begin testing. 

**SCP Fee Fix** 
- Waiting on Deanna to respond

**Comment by TracyLinder** [comment]
See #2985 for PSM testing updates.

**Comment by TracyLinder** [comment]
- [ ] Tell Unisys about PSM Fee Issue resolution/feedback from Paloma
- [ ] Send Unisys PSM email templates
- [ ] Finish post payment bugs
- [ ] Schedule SCP3 UAT

**Comment by TracyLinder** [comment]
### 7/14

**Unisys:**
- Haven't heard back from the promote requests w/ CTM
    - LM Map viewer
    - PSM fee/info fix
    - UC/LM self-assign (into DEV)
- SCP3 promote to Prod
    - David needs to send me an email for feedback on a couple of possible issues
- David sent ROWMAN invoice to Del & copied me

**Tracy**
- [ ] See unfinished tasks above
- ROWMAN 4 SOW (for scoping already done) was approved

**Comment by TracyLinder** [comment]
### 7/21

- VPN access fixed today
- Paul will be verifying promotes & they'll let us know
- Tracy still needs to finish work on Post Payment bugs & PSM email content

**Comment by TracyLinder** [comment]
###7/28

- Should be able to do all SCP3 testing after Tuesday of next week. 
- Pending promotes (in CTM queue): UC/LM Self Assign, PSM fee fix, SCP3 
- Unisys is working on the LM Click Map updates requested yesterday

**Comment by TracyLinder** [comment]
### 8/4

**Unisys:**
- LM Click Map has been promoted to Test
- Remove # of parking spaces from SW Cafe in test has promoted to test
- Remaining:
    - PSM email content
    - UC/LM Self-Assignment Changes
    - The last of SCP3 is pending promote

**ATD**
- [ ] Tracy needs to finish post payment things

**Comment by TracyLinder** [comment]
### 8/11

Unisys:
- Map Viewer: UC & LM need to be tested
- PSM: Pending content to be promoted (emails), Fee/info fix needs to be tested
- Self Assign: David & I went through it last week. We can start testing this now
- [ ] David to confirm if SCP3 was promoted or not


Tracy
- [ ] Schedule LM map UAT
- [ ] Test PSM fee fix
- [ ] Finish post payment things
- [ ] Schedule UC/LM self-assign UAT

**Comment by TracyLinder** [comment]
### 10/27/20

- Post Payment Bugs: Pending Promote
- PSM: 2 issues pending promote (Tracy will reschedule UAT)

**Comment by TracyLinder** [comment]
### 11/3
- PSM: All items have been promoted; UAT scheduled 11/10
- Post payment: All items promoted; Ready for UAT
- Self assignment will promote on Thursday 11/5
- SCP3: Fixes are in test; UAT ready

**Comment by TracyLinder** [comment]
### 11/17/20
- Still waiting on promote date on PSM (waiting on CTM)
- Post payment deploy to PROD will probably happen Thursday
- [ ] Ask Gita to invite me to meeting tomorrow re: developer meeting about the freeze
- RW TURP issue: Paul is looking into it (possible a wrong table was promoted)
- SCP3: There are a couple of defects due to commits not making it to PROD (pending George w/ CTM to respond)

**Comment by TracyLinder** [comment]
### 11/24/20

**ROWMAN Projects**

- PSM in prod, no issues reported yet
- Post payment all in PROD

The warranty period is 30 days from promote. So the deadline is 12/20/20. 
Del will send us a close-out report that includes phase 2 & 3. 

**SCP**
- There are issues w/ older folders that don't have the new functionality. 
    - Re-defaulting will fix the info tab; David is taking care of it
